docs/src/app/components/pages/components/RaisedButton/Page.jsx

```diff
@@ -11,12 +11,17 @@ import raisedButtonExampleSimpleCode from '!raw!./ExampleSimple';
 import RaisedButtonExampleSimple from './ExampleSimple';
 import raisedButtonExampleComplexCode from '!raw!./ExampleComplex';
 import RaisedButtonExampleComplex from './ExampleComplex';
+import raisedButtonExampleIconCode from '!raw!./ExampleIcon';
+import RaisedButtonExampleIcon from './ExampleIcon';
 
 const descriptions = {
   simple: '`RaisedButton` with default color, `primary`, `secondary` and and `disabled` props applied.',
   complex: 'The first example uses an `input` as a child component, ' +
   'the next has next has an [SVG Icon](/#/components/svg-icon), with the label positioned after. ' +
   'The final example uses a [Font Icon](/#/components/font-icon), and is wrapped in an anchor tag.',
+  icon: 'Example uses a [SVG Icon](/#/components/svg-icon) without label. First button has default' +
```

Comment:
'Example using icons without a label. The first example uses an [SVG Icon](/#/components/svg-icon), and has the default color.'